status-im/status-react
# bug #4508 - removes tokens and fiat currencies unsupported by cryptoc…

fixes #4508 

### Summary:

Removes a number of tokens and currencies which are unsupported by cryptocompare. Includes a migration that sets the selected currency in Profile to USD if the previous currency was among the removed ones. Also includes a migration that deselects removed tokens from user's selection of tokens in Wallet.

"Removed" tokens are merely removed from the selection and then hidden from the list of all possible assets. They are not deleted from the code because we still want to show any previous token transactions in the history even if we can't tell their fiat value.

### Testing notes (optional):

Keep an eye on migrations. Specific cases are:
- when currency in Profile was one of the removed ones (should be set to USD by migration)
- when currency was not among removed ones (should remain as it was)
- when user had removed tokens in their selection on main Wallet screen (should be removed from both selection and Manage Assets)
- when user had transactions with said tokens in Transaction history (transactions should still appear)

### Steps to test:
- Open Wallet and check the Token lists
- Open Profile and check the fiat currencies lists


status: ready
